pennylane/tape/tape.py

```diff
@@ -483,3 +484,6 @@ def __setitem__(self, key, val):
 
     def __hash__(self):
         return QuantumScript.__hash__(self)
+
+
+register_pytree(QuantumTape, QuantumTape._flatten, QuantumTape._unflatten)
```

Comment:
Do we care about doing this for `OperationRecorder`?

Reply:
Since `OperationRecorder` is more about "What did I queue?" I don't think we need to pay too much attention to parameter management on it.